### ISSUE TITLE
System UI crash (?)

### ISSUE THREAD

**Issue by mverboom** [chenxiaolong/RSAF#40]
Hi,

Did some testing with different backends (cifs and ssh), and functionality seems to work great.
But when I let an application use SAF to access files (for example Poweramp with a share with lots of music),
it starts working nicely but then the system does a soft restart (UI restart?).

I understand this is quite a vague problem description. Would really like to get this working, functionally it is
great and I also see pretty nice speeds.

If this isn't a known issue, is there something I can do to debug?

Regards,

Mark

**Comment by chenxiaolong** [comment]
Wow, that's an interesting issue. It's weird that an unprivileged app like RSAF would be able to cause `SystemUI` to crash like that.

It'll probably be difficult to obtain the relevant logs after the crash. `SystemUI` restarting will likely produce enough log messages to push out the errors from the crash.

Is the problem easily reproducible on your device? If so, can you start a logcat session with `adb logcat > logcat.txt`, reproduce the crash, and then upload `logcat.txt`? (If you prefer not to post it publicly, feel free to email it to me at `accounts+github <AT> chiller3.com`.)

**Comment by mverboom** [comment]
Thanks for the reply. Before opening the issue I disabled RSAF and phone was ok again.
After reading your reply I created new remotes and let poweramp index the whole mp3 directory with a logcat
running. Interestingly, everything was ok! Then added another remote and tried to backup my phone to it and it
also was fine (and getting pretty nice speeds).
The new remotes I created were ssh based. During the initial problem I used cifs. So created cifs remotes and tried again, everything fine again.

I don't know what happened, maybe those cosmic rays again?

Very happy with the results at the moment, best SAF provider I've found after having struggled with DAVx5's webdav and CIFS Document Provider.

Regards,

Mark
